iaso/tests/api/test_setup_account.py

```diff
@@ -122,3 +122,17 @@ def test_setup_account_has_all_perms(self):
         self.assertEqual(m.Profile.objects.filter(user__username="unittest_username").count(), 1)
         self.assertEqual(m.User.objects.filter(username="unittest_username").count(), 1)
         self.assertEqual(has_all_perms, True)
+
+    def test_setup_account_project_creation(self):
+        self.client.force_authenticate(self.admin)
+
+        data = {
+            "account_name": "initial_project_account",
+            "user_username": "username",
+            "password": "password",
+        }
+
+        response = self.client.post("/api/setupaccount/", data=data, format="json")
+        self.assertEqual(response.status_code, 201)
+        project = m.Project.objects.filter(name="initial_project_account project")
+        self.assertEqual(len(project), 1)
```

Comment:
This is not testing that the project has the correct app_id, that it's linked to the correct account, or that it is linked to the proper data source.